docs/installation/mac.md

```diff
@@ -232,7 +317,7 @@ Work through this section to try some practical container tasks using `boot2dock
 
 ### Mount a volume on the container
 
-When you start `boot2docker`, it automatically shares your `/Users` directory
+When you start a container it automatically shares your `/Users` directory
```

Comment:
This PR on this in machine. https://github.com/docker/machine/pull/1552 --- I'll tweak that